### ISSUE TITLE
Placeholder has wrong font size when UITextView is prepopulated with text

### ISSUE THREAD

**Issue by mattlogan** [devxoul/UITextView-Placeholder#49]
I have a UITextView with font size 22, specified in interface builder. I have the following code in `viewDidLoad()`.

```
textView.placeHolder = "Placeholder text"
textView.text = "My real text"
```

The UITextView appears with "My real text" at the correct font size, but when I delete this text to reveal the placeholder, the placeholder font size appears to be 17 (or some smaller default).

**Comment by devxoul** [comment]
Hmm, can you share your project? It works on my machine. I tested with the demo project.

```objc
// ViewController.h
@property (nonatomic, strong) IBOutlet UITextView *textView;

// ViewController.m
- (void)viewDidLoad {
    [super viewDidLoad];
    self.textView.placeholder = @"Placeholder text";
    self.textView.text = @"My real text";
}
```

<img width="1118" alt="screenshot" src="https://user-images.githubusercontent.com/931655/40282673-dda41682-5cad-11e8-827e-448a3fee0cf2.png">

**Comment by mattlogan** [comment]
I was able to fix my issue by setting the placeholder text before setting the TextView's text. You can reproduce by changing the above code to:

```
// ViewController.m
- (void)viewDidLoad {
    [super viewDidLoad];
    self.textView.text = @"My real text";
    self.textView.placeholder = @"Placeholder text";
}
```

Not sure if this is a bug worth fixing but it might be worth documenting! Thanks for the response.

**Comment by devxoul** [comment]
I could reproduce the bug with your code. #49 is a patch for the issue. Could you check this? Thanks!

**Comment by mattlogan** [comment]
I'm not sure this is the right fix. When I apply your patch, I'm now seeing the text overlapping the placeholder view:

<img width="219" alt="screenshot 2018-05-21 10 12 41" src="https://user-images.githubusercontent.com/4635072/40320153-84c12ece-5cdf-11e8-98d7-73e1fff84d89.png">

**Comment by devxoul** [comment]
I should have written much tests... 🤦‍♂️ I have fixed the PR, could you please check it again?

**Comment by mattlogan** [comment]
This looks good! I verified it in my project.

**Comment by devxoul** [comment]
Thanks for quick check! I released [1.2.1](https://github.com/devxoul/UITextView-Placeholder/releases/tag/1.2.1) to the CocoaPods and it will be available with `$ pod update` command :tada:
